docs/getting-started/advanced-setting.asciidoc

```diff
@@ -175,3 +175,52 @@ By default, Elastic prebuilt rules in the *Rules* and *Rule Monitoring* tables i
 == Manage alert tag options
 
 The `securitySolution:alertTags` field determines which options display in the alert tag menu. The default alert tag options are `Duplicate`, `False Positive`, and `Further investigation required`. You can update the alert tag menu by editing these options or adding more. To learn more about using alert tags, refer to <<apply-alert-tags>>.
+
+[discrete]
+[[exclude-cold-frozen-data-rule-executions]]
+== Exclude cold and frozen data from rule executions 
+
+To ensure rules don't search cold and frozen data when executing, specify cold and frozen {ref}/data-tiers.html[data tiers] in the `excludedDataTiersForRuleExecution` field. Multiple data tiers must be separated by commas, for example: `data_frozen`, `data_cold`. This setting is turned off by default; turning it on can improve rule performance and reduce execution time. 
+
+The `excludedDataTiersForRuleExecution` setting does not apply to {esql} or {ml} rules. 
+
+[TIP]
+====
+
+The `excludedDataTiersForRuleExecution` setting applies to all rules in a {kib} space. To only exclude cold and frozen data from specific rules, add a {ref}/sql-rest-filtering.html[Query DSL] filter to the rules you want affected. Add the filter when creating a new rule or updating an existing one. 
+
+Here is a sample Query DSL filter that excludes frozen tier data from a rule's execution:
+
+[source,console]
+----
+{
+   "bool":{
+      "must_not":{
+         "terms":{
+            "_tier":[
+               "data_frozen"
+            ]
+         }
+      }
+   }
+}
+----
+
+Here is another sample Query DSL filter that excludes cold and frozen tier data from a rule's execution:
+
+[source,console]
+----
+{
+   "bool":{
+      "must_not":{
+         "terms":{
+            "_tier":[
+               "data_frozen", "data_cold"
+            ]
+         }
+      }
+   }
+}
+----
```

Comment:
These example queries seem helpful, but a little out of place here. This topic is about what the user can do in the Advanced Settings UI to change system-wide settings, not how to create or edit an individual rule. The examples might fit better in the rule creation docs, maybe even as a short sub-topic of the main "Create a rule" page (since that page itself is already so long).

Reply:
Agree that this isn't an ideal location for this information. I only put it here to keep related information together. That said, I did consider creating a sub-topic under the main "Create a rule" page, but wasn't sure if there'd be enough content to justify it. I'll try creating a new topic to see if it works.

Reply:
Yeah, it's kinda related, but kinda not. There might not be a better spot for it, so here's fine if a separate topic doesn't work.

Reply:
I [created a new topic](https://github.com/elastic/security-docs/pull/5849/commits/4a6659780a395f7fc6dd7e3400316bb7616ba1ba) for the approach that only updates individual rules. @vitaliidm and @joepeeples would you mind taking a look at the modified draft once the CI checks complete?